common/src/main/java/org/jboss/hal/testsuite/creaper/ModelNodeUtils.java

```diff
@@ -0,0 +1,27 @@
+package org.jboss.hal.testsuite.creaper;
```

Comment:
I would suggest to move this class to the `org.jboss.hal.testsuite.dmr` package (there is already e.g. `ModelNodeGenerator` in this package)